### PR TITLE
perf: make it run faster

### PR DESCRIPTION
**Pull request by alpheratz0**
* remove multiple calls to jq
* remove multiple calls to grep, tail and cut
* remove all subshells

### CODE REVIEW

File: btc [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-ChangeDay=$(curl_data \"CHANGEDAY\" 25-30)
-Supply=$(curl_data \"SUPPLY\" 23-34)
-MktCap=$(curl_data \"MKTCAP\" 22-29)
-
-echo "The current price of BTC is: $"$price "USD
-			     "$ChangePCTDay"% daily change
-		   	     $"$ChangeDay "USD daily change
-		 	     $"$Supply "USD total supply
-			     $"$MktCap "USD" market cap
+curl -s "https://min-api.cryptocompare.com/data/pricemultifull?fsyms=BTC&tsyms=USD&e=CCCAGG" |
```
Comment (Edesem) [inline btc:3]: Damn yeah this is pretty good. Great stuff! It works way quicker and only uses 2 commands!

**Review by Edesem — APPROVED** [review]
Wow, this destroys my original setup. and gets and average of 0.03 s compared to roughly 0.17 s of my original script.

**Comment by alpheratz0** [comment]
> Wow, this destroys my original setup. and gets and average of 0.03 s compared to roughly 0.17 s of my original script.

;) you should check `man jq`, it is long but it is also a very useful tool

**Comment by alpheratz0** [comment]
@Edesem you have a typo, it's not GLP it is GPL, also I prefer if you @ my github instead of my youtube channel :), thanks

**Comment by Edesem** [comment]
> @Edesem you have a typo, it's not GLP it is GPL, also I prefer if you @ my github instead of my youtube channel :), thanks

Fixed :)

**Pull request merged**